nemo_skills/dataset/flores200/prepare.py

```
@@ -0,0 +1,66 @@
+from langcodes import Language
+
+from datasets import load_dataset
+from tqdm import tqdm
+
+
+def write_data_to_file(output_file, datasets, src_languages, tgt_languages):
+    with open(output_file, "wt", encoding="utf-8") as fout:
+        for src_lang in src_languages:
+            for tgt_lang in tgt_languages:
+                if src_lang != tgt_lang:
+                    for src, tgt in zip(datasets[src_lang], datasets[tgt_lang]):
```

Ruff (0.13.1)

`zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)